src/accounts.rs

```diff
@@ -301,36 +302,61 @@ impl Accounts {
     ///
     /// This is an auxiliary function and not part of public API.
     /// Use [Accounts::background_fetch] instead.
-    async fn background_fetch_without_timeout(&self) {
+    async fn background_fetch_without_timeout(accounts: Vec<Context>, events: Events) {
```

Comment:
This is `background_fetch_without_timeout()`, the next one is `background_fetch_with_timeout()`. Maybe i've had too much coffee, but i had to read it several times to see the difference :) I think the `_without_timeout` suffix could be better removed

Reply:
Better `_no_timeout` then. `_without_timeout` looks very similar to `_with_timeout`